src/placement/pl_map.c

```diff
@@ -275,7 +275,7 @@ struct d_hash_table	pl_htable = {
 
 #define DSR_RING_DOMAIN		PO_COMP_TP_RACK
 
-static void
+void
 pl_map_attr_init(struct pool_map *po_map, pl_map_type_t type,
```

Comment:
Just a general comment unrelated to your patch.   We should use const in function prototypes for complex things that don't change, in this case, po_map

Reply:
I will update it when refresh the patch next time.